src/runner.rs

```diff
@@ -128,3 +128,104 @@ impl Runner for FirefoxRunner {
         Ok(self.ret_code)
     }
 }
+
+pub fn find_binary(name: &str) -> Option<PathBuf> {
```

Comment:
Do you need to make it `pub` if it is to be consumed by the modules below? If you don’t, I think we should make un-publish this.